src/proxies/panel_manager.js

```diff
@@ -15,7 +16,9 @@ PanelManager.setPoi = async function(poi, options) {
     if(panel.isPoiComplient) {
       panel.setPoi(poi, options)
     } else {
-      panel.close()
+      if(options && !options.isFromList){
+        panel.close()
```

Comment:
what's the expected behavior here ?  
This causes the favorite panel to stay active when clicking on a POI on the map, which is probably unexpected.

Reply:
It's expected to close the panel unless we click on a PJ list item...
I don't see any problem with the favorite panel. do you mean the category/ies panel?
Please detail the buggy scenario, I don't see anything wrong yet